### ISSUE TITLE
Crash on startup

### ISSUE THREAD

**Issue by MrGrapey** [AllTheMods/ATM3-Lite#21]
Everything worked fine last night, today it crashes everytime phase 2 for extra utilities comes up. Crashing for friends aswell. (has been working for weeks?)

---- Minecraft Crash Report ----

WARNING: coremods are present:
  ShetiPhian-ASM (ShetiPhian-ASM-1.12.0.jar)
  ForgelinPlugin (Forgelin-1.7.4.jar)
  BedPatch (bedpatch-2.2-1.12.2.jar)
  EnderCorePlugin (EnderCore-1.12.2-0.5.31.jar)
  LoadingPlugin (Quark-r1.4-125.jar)
  Inventory Tweaks Coremod (InventoryTweaks-1.64-dev.jar)
  AstralCore (astralsorcery-1.12.2-1.9.4.jar)
  CTMCorePlugin (CTM-MC1.12-0.3.0.15.jar)
  CoreMod (Aroma1997Core-1.12.2-2.0.0.0.b155.jar)
  IELoadingPlugin (ImmersiveEngineering-core-0.12-82.jar)
  LoadingPlugin (ResourceLoader-MC1.12.1-1.5.3.jar)
  Do not report to Forge! Remove FoamFixAPI (or replace with FoamFixAPI-Lawful) and try again. (foamfix-0.9.9.1-1.12.2-anarchy.jar)
Contact their authors BEFORE contacting forge

// There are four lights!

Time: 8/16/18 2:22 PM
Description: There was a severe problem during mod loading that has caused the game to fail

net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from Extra Utilities 2 (extrautils2)
Caused by: java.lang.ExceptionInInitializerError
	at com.rwtema.extrautils2.ExtraUtils2$1.run(ExtraUtils2.java:153)
	at com.rwtema.extrautils2.XUProxyClient.run(XUProxyClient.java:54)
	at com.rwtema.extrautils2.ExtraUtils2.init(ExtraUtils2.java:149)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:626)
	at sun.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91)
	at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150)
	at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76)
	at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
	at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71)
	at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116)
	at com.google.common.eventbus.EventBus.post(EventBus.java:217)
	at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:218)
	at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:196)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91)
	at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150)
	at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76)
	at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
	at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71)
	at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116)
	at com.google.common.eventbus.EventBus.post(EventBus.java:217)
	at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:135)
	at net.minecraftforge.fml.common.Loader.initializeMods(Loader.java:744)
	at net.minecraftforge.fml.client.FMLClientHandler.finishMinecraftLoading(FMLClientHandler.java:329)
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:534)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:377)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
Caused by: java.lang.NullPointerException
	at com.rwtema.extrautils2.book.BookHandler$1.func_110549_a(BookHandler.java:47)
	at net.minecraft.client.resources.SimpleReloadableResourceManager.func_110542_a(SimpleReloadableResourceManager.java:121)
	at com.rwtema.extrautils2.book.BookHandler.<clinit>(BookHandler.java:32)
	... 43 more

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.12.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 1111747608 bytes (1060 MB) / 2360344576 bytes (2251 MB) up to 2863661056 bytes (2731 MB)
	JVM Flags: 4 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx3072m -Xms256m -XX:PermSize=256m
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP 9.42 Powered by Forge 14.23.4.2712 141 mods loaded, 141 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored

	| State | ID                                | Version                      | Source                                              | Signature                                |
	|:----- |:--------------------------------- |:---------------------------- |:--------------------------------------------------- |:---------------------------------------- |
	| UCHI  | minecraft                         | 1.12.2                       | minecraft.jar                                       | None                                     |
	| UCHI  | mcp                               | 9.42                         | minecraft.jar                                       | None                                     |
	| UCHI  | FML                               | 8.0.99.99                    | forge-1.12.2-14.23.4.2712.jar                       | e3c3d50c7c986df74c645c0ac54639741c90a557 |
	| UCHI  | forge                             | 14.23.4.2712                 | forge-1.12.2-14.23.4.2712.jar                       | e3c3d50c7c986df74c645c0ac54639741c90a557 |
	| UCHI  | foamfixcore                       | 7.7.4                        | minecraft.jar                                       | None                                     |
	| UCHI  | simplebedrolls                    | 1.2.1                        | [1.12]SimpleBedrolls-1.2.1.jar                      | None                                     |
	| UCHI  | fastbench                         | 1.5.0                        | FastWorkbench-1.12.2-1.5.0.jar                      | None                                     |
	| UCHI  | actuallyadditions                 | 1.12.2-r140                  | ActuallyAdditions-1.12.2-r140.jar                   | None                                     |
	| UCHI  | baubles                           | 1.5.2                        | Baubles-1.12-1.5.2.jar                              | None                                     |
	| UCHI  | actuallybaubles                   | 1.1                          | ActuallyBaubles-1.12-1.1.jar                        | None                                     |
	| UCHI  | adventurepack                     | 1.5.0                        | adventurepack-1.5.jar                               | None                                     |
	| UCHI  | akashictome                       | 1.2-10                       | AkashicTome-1.2-10.jar                              | None                                     |
	| UCHI  | ctm                               | MC1.12-0.3.0.15              | CTM-MC1.12-0.3.0.15.jar                             | None                                     |
	| UCHI  | chisel                            | MC1.12.2-0.2.0.31            | Chisel-MC1.12.2-0.2.0.31.jar                        | None                                     |
	| UCHI  | endercore                         | 1.12.2-0.5.31                | EnderCore-1.12.2-0.5.31.jar                         | None                                     |
	| UCHI  | jei                               | 4.10.0.200                   | jei_1.12.2-4.10.0.200.jar                           | None                                     |
	| UCHI  | thaumcraft                        | 6.1.BETA21                   | Thaumcraft-1.12.2-6.1.BETA21.jar                    | None                                     |
	| UCHI  | codechickenlib                    | 3.2.0.345                    | CodeChickenLib-1.12.2-3.2.0.345-universal.jar       | f1850c39b2516232a2108a7bd84d1cb5df93b261 |
	| UCHI  | redstoneflux                      | 2.0.2                        | RedstoneFlux-1.12-2.0.2.3-universal.jar             | 8a6abf2cb9e141b866580d369ba6548732eff25f |
	| UCHI  | cofhcore                          | 4.5.2                        | CoFHCore-1.12.2-4.5.2.19-universal.jar              | None                                     |
	| UCHI  | cofhworld                         | 1.2.0                        | CoFHWorld-1.12.2-1.2.0.5-universal.jar              | 8a6abf2cb9e141b866580d369ba6548732eff25f |
	| UCHI  | thermalfoundation                 | 2.5.0                        | ThermalFoundation-1.12.2-2.5.0.19-universal.jar     | 8a6abf2cb9e141b866580d369ba6548732eff25f |
	| UCHI  | thermalexpansion                  | 5.5.0                        | ThermalExpansion-1.12.2-5.5.0.29-universal.jar      | 8a6abf2cb9e141b866580d369ba6548732eff25f |
	| UCHI  | enderio                           | 5.0.26                       | EnderIO-1.12.2-5.0.26.jar                           | None                                     |
	| UCHI  | enderiointegrationtic             | 5.0.26                       | EnderIO-1.12.2-5.0.26.jar                           | None                                     |
	| UCHI  | mantle                            | 1.12-1.3.2.24                | Mantle-1.12-1.3.2.24.jar                            | None                                     |
	| UCHI  | tconstruct                        | 1.12.2-2.10.1.87             | TConstruct-1.12.2-2.10.1.87.jar                     | None                                     |
	| UCHEE | extrautils2                       | 1.0                          | extrautils2-1.12-1.7.6.jar                          | None                                     |
	| UCH   | flyringbaublemod                  | 0.3.1_1.12-d4e654e           | angelRingToBauble-1.12-0.3.1.50+d4e654e.jar         | None                                     |
	| UCH   | appleskin                         | 1.0.9                        | AppleSkin-mc1.12-1.0.9.jar                          | None                                     |
	| UCH   | aroma1997core                     | 2.0.0.0.b155                 | Aroma1997Core-1.12.2-2.0.0.0.b155.jar               | dfbfe4c473253d8c5652417689848f650b2cbe32 |
	| UCH   | aroma1997sdimension               | 2.0.0.2.b77                  | Aroma1997s-Dimensional-World-1.12.2-2.0.0.2.b77.jar | dfbfe4c473253d8c5652417689848f650b2cbe32 |
	| UCH   | crafttweaker                      | 4.1.9                        | CraftTweaker2-1.12-4.1.9.jar                        | None                                     |
	| UCH   | astralsorcery                     | 1.9.4                        | astralsorcery-1.12.2-1.9.4.jar                      | a0f0b759d895c15ceb3e3bcb5f3c2db7c582edf0 |
	| UCH   | morphtool                         | 1.2-16                       | Morph-o-Tool-1.2-16.jar                             | None                                     |
	| UCH   | quark                             | r1.4-125                     | Quark-r1.4-125.jar                                  | None                                     |
	| UCH   | autoreglib                        | 1.3-18                       | AutoRegLib-1.3-18.jar                               | None                                     |
	| UCH   | badwithernocookiereloaded         | 1.12.2-3.1.14                | badwithernocookiereloaded-1.12.2-3.1.14.jar         | None                                     |
	| UCH   | base                              | 3.9.0                        | base-1.12.2-3.9.0.jar                               | None                                     |
	| UCH   | betterbuilderswands               | 0.11.1                       | BetterBuildersWands-1.12-0.11.1.245+69d0d70.jar     | None                                     |
	| UCH   | bookshelf                         | 2.3.552                      | Bookshelf-1.12.2-2.3.552.jar                        | d476d1b22b218a10d845928d1665d45fce301b27 |
	| UCH   | botania                           | r1.10-354                    | Botania r1.10-354.jar                               | None                                     |
	| UCH   | ceramics                          | 1.12-1.3.4                   | Ceramics-1.12-1.3.4.jar                             | None                                     |
	| UCH   | chameleon                         | 1.12-4.1.3                   | Chameleon-1.12-4.1.3.jar                            | None                                     |
	| UCH   | clumps                            | 3.1.1                        | Clumps-3.1.1.jar                                    | None                                     |
	| UCH   | cyclopscore                       | 0.11.6                       | CyclopsCore-1.12.2-0.11.6.jar                       | bd0353b3e8a2810d60dd584e256e364bc3bedd44 |
	| UCH   | commoncapabilities                | 1.4.0                        | CommonCapabilities-1.12-1.4.0.jar                   | None                                     |
	| UCH   | conarm                            | 1.0.0                        | conarm-1.12.2-1.0.0.jar                             | 5d5b8aee896a4f5ea3f3114784742662a67ad32f |
	| UCH   | controlling                       | 3.0.6                        | Controlling-3.0.6.jar                               | None                                     |
	| UCH   | craftingtweaks                    | 8.1.9                        | CraftingTweaks_1.12.2-8.1.9.jar                     | None                                     |
	| UCH   | ctgui                             | 1.0.0                        | CraftTweaker2-1.12-4.1.9.jar                        | None                                     |
	| UCH   | crafttweakerjei                   | 2.0.2                        | CraftTweaker2-1.12-4.1.9.jar                        | None                                     |
	| UCH   | custommainmenu                    | 2.0.8                        | CustomMainMenu-MC1.12.2-2.0.8.jar                   | None                                     |
	| UCH   | waila                             | 1.8.26                       | Hwyla-1.8.26-B41_1.12.2.jar                         | None                                     |
	| UCH   | p455w0rdslib                      | 2.0.29                       | p455w0rdslib-1.12-2.0.29.jar                        | None                                     |
	| UCH   | stg                               | 1.12.2-1.2.3                 | stg-1.12.2-1.2.3.jar                                | None                                     |
	| UCH   | danknull                          | 1.4.33                       | DankNull-1.12-1.4.33.jar                            | None                                     |
	| UCH   | diethopper                        | 1.1                          | diethopper-1.1.jar                                  | None                                     |
	| UCH   | dismantler                        | 0.1.2                        | dismantler-0.1.2.jar                                | None                                     |
	| UCH   | elevatorid                        | 1.3.6                        | ElevatorMod-1.12.2-1.3.6.jar                        | None                                     |
	| UCH   | csb_ench_table                    | 1.1.3                        | EnchantingTable-1.12-1.1.3.jar                      | None                                     |
	| UCH   | enderiobase                       | 5.0.26                       | EnderIO-1.12.2-5.0.26.jar                           | None                                     |
	| UCH   | enderioconduits                   | 5.0.26                       | EnderIO-1.12.2-5.0.26.jar                           | None                                     |
	| UCH   | enderioconduitsappliedenergistics | 5.0.26                       | EnderIO-1.12.2-5.0.26.jar                           | None                                     |
	| UCH   | storagedrawers                    | 1.12-5.3.5                   | StorageDrawers-1.12.2-5.3.7.jar                     | None                                     |
	| UCH   | refinedstorage                    | 1.5.34                       | refinedstorage-1.5.34.jar                           | 57893d5b90a7336e8c63fe1c1e1ce472c3d59578 |
	| UCH   | enderioconduitsopencomputers      | 5.0.26                       | EnderIO-1.12.2-5.0.26.jar                           | None                                     |
	| UCH   | enderioconduitsrefinedstorage     | 5.0.26                       | EnderIO-1.12.2-5.0.26.jar                           | None                                     |
	| UCH   | enderiointegrationforestry        | 5.0.26                       | EnderIO-1.12.2-5.0.26.jar                           | None                                     |
	| UCH   | enderiointegrationticlate         | 5.0.26                       | EnderIO-1.12.2-5.0.26.jar                           | None                                     |
	| UCH   | ftblib                            | 5.2.0.23                     | FTBLib-5.2.0.23.jar                                 | None                                     |
	| UCH   | enderiomachines                   | 5.0.26                       | EnderIO-1.12.2-5.0.26.jar                           | None                                     |
	| UCH   | enderiopowertools                 | 5.0.26                       | EnderIO-1.12.2-5.0.26.jar                           | None                                     |
	| UCH   | enderstorage                      | 2.4.3.131                    | EnderStorage-1.12.2-2.4.3.131-universal.jar         | f1850c39b2516232a2108a7bd84d1cb5df93b261 |
	| UCH   | shetiphiancore                    | 3.5.7                        | shetiphiancore-1.12.0-3.5.7.jar                     | None                                     |
	| UCH   | endertanks                        | 1.6.6                        | endertanks-1.12.0-1.6.6.jar                         | None                                     |
	| UCH   | valkyrielib                       | 1.12.2-2.0.11a               | valkyrielib-1.12.2-2.0.11a.jar                      | None                                     |
	| UCH   | environmentaltech                 | 1.12.2-2.0.11a               | environmentaltech-1.12.2-2.0.11a.jar                | None                                     |
	| UCH   | etlunar                           | 1.12.2-2.0.11a               | etlunar-1.12.2-2.0.11a.jar                          | None                                     |
	| UCH   | shadowmc                          | 3.8.0                        | ShadowMC-1.12-3.8.0.jar                             | None                                     |
	| UCH   | extrarails                        | 1.3.0                        | ExtraRails-1.12-1.3.0.jar                           | None                                     |
	| UCH   | foamfix                           | 0.9.9.1-1.12.2               | foamfix-0.9.9.1-1.12.2-anarchy.jar                  | None                                     |
	| UCH   | forgelin                          | 1.7.4                        | Forgelin-1.7.4.jar                                  | None                                     |
	| UCH   | journeymap                        | 1.12.2-5.5.2                 | journeymap-1.12.2-5.5.2.jar                         | None                                     |
	| UCH   | ftbutilities                      | 5.2.0.28                     | FTBUtilities-5.2.0.28.jar                           | None                                     |
	| UCH   | fw                                | 1.6.0                        | FullscreenWindowed-1.12-1.6.0.jar                   | None                                     |
	| UCH   | guideapi                          | 1.12-2.1.5-60                | Guide-API-1.12-2.1.5-60.jar                         | None                                     |
	| UCH   | immersiveengineering              | 0.12-82                      | ImmersiveEngineering-0.12-82.jar                    | 4cb49fcde3b43048c9889e0a3d083225da926334 |
	| UCH   | immersivepetroleum                | 1.1.9                        | immersivepetroleum-1.12.2-1.1.9.jar                 | None                                     |
	| UCH   | mekanism                          | 1.12.2-9.4.13.349            | Mekanism-1.12.2-9.4.13.349.jar                      | None                                     |
	| UCH   | teslacorelib                      | 1.0.14                       | tesla-core-lib-1.12.2-1.0.14.jar                    | d476d1b22b218a10d845928d1665d45fce301b27 |
	| UCH   | industrialforegoing               | 1.12.2-1.12.2                | industrialforegoing-1.12.2-1.10.1-176.jar           | None                                     |
	| UCH   | industrialwires                   | 1.6-25                       | IndustrialWires-1.6-25.jar                          | 7e11c175d1e24007afec7498a1616bef0000027d |
	| UCH   | mtlib                             | 3.0.5                        | MTLib-3.0.5.jar                                     | None                                     |
	| UCH   | initialinventory                  | 2.0.2                        | InitialInventory-3.0.0.jar                          | None                                     |
	| UCH   | integrateddynamics                | 0.11.13                      | IntegratedDynamics-1.12.2-0.11.13.jar               | bd0353b3e8a2810d60dd584e256e364bc3bedd44 |
	| UCH   | integrateddynamicscompat          | 1.0.0                        | IntegratedDynamics-1.12.2-0.11.13.jar               | None                                     |
	| UCH   | integratedtunnels                 | 1.5.6                        | IntegratedTunnels-1.12.2-1.5.6.jar                  | bd0353b3e8a2810d60dd584e256e364bc3bedd44 |
	| UCH   | integratedtunnelscompat           | 1.0.0                        | IntegratedTunnels-1.12.2-1.5.6.jar                  | None                                     |
	| UCH   | natura                            | 1.12.2-4.3.2.49              | natura-1.12.2-4.3.2.49.jar                          | None                                     |
	| UCH   | silentlib                         | 2.3.2                        | SilentLib-1.12.2-2.3.2-109.jar                      | None                                     |
	| UCH   | silentgems                        | 2.7.2                        | SilentsGems-1.12.2-2.7.2b-270.jar                   | None                                     |
	| UCH   | integrationforegoing              | 1.12.2-1.7                   | IntegrationForegoing-1.12.2-1.7.jar                 | 4ffa87db52cf086d00ecc4853a929367b1c39b5c |
	| UCH   | inventorytweaks                   | 1.64-dev+release.110.b4fac73 | InventoryTweaks-1.64-dev.jar                        | 55d2cd4f5f0961410bf7b91ef6c6bf00a766dcbe |
	| UCH   | ironbackpacks                     | 1.12.2-3.0.8-12              | IronBackpacks-1.12.2-3.0.8-12.jar                   | None                                     |
	| UCH   | ironchest                         | 1.12.2-7.0.44.829            | ironchest-1.12.2-7.0.44.829.jar                     | None                                     |
	| UCH   | itemscroller                      | 0.11.0                       | itemscroller-1.12.0-0.11.0.jar                      | None                                     |
	| UCH   | jeiintegration                    | 1.5.1.36                     | jeiintegration_1.12.2-1.5.1.36.jar                  | None                                     |
	| UCH   | jeresources                       | 0.8.7.41                     | JustEnoughResources-1.12.2-0.8.7.41.jar             | None                                     |
	| UCH   | llor                              | 1.1.6-mc1.12.2               | LLOverlayReloaded-1.1.6-mc1.12.2.jar                | None                                     |
	| UCH   | mcjtylib_ng                       | 3.0.2                        | mcjtylib-1.12-3.0.2.jar                             | None                                     |
	| UCH   | mekanismgenerators                | 9.4.11                       | MekanismGenerators-1.12.2-9.4.13.349.jar            | None                                     |
	| UCH   | mekanismtools                     | 9.4.11                       | MekanismTools-1.12.2-9.4.13.349.jar                 | None                                     |
	| UCH   | netherportalfix                   | 5.3.13                       | NetherPortalFix_1.12.1-5.3.13.jar                   | None                                     |
	| UCH   | oreexcavation                     | 1.4.119                      | OreExcavation-1.4.119.jar                           | None                                     |
	| UCH   | oeintegration                     | 2.3.3                        | oeintegration-2.3.3.jar                             | None                                     |
	| UCH   | oldjava                           | 1.0.2                        | OldJavaWarning-1.12.2-1.0.2.jar                     | d476d1b22b218a10d845928d1665d45fce301b27 |
	| UCH   | openglider                        | @VERSION@                    | OpenGlider-1.12.1-1.1.0.jar                         | None                                     |
	| UCH   | placebo                           | 1.4.0                        | Placebo-1.12.2-1.4.0.jar                            | None                                     |
	| UCH   | quickleafdecay                    | 1.2.4                        | QuickLeafDecay-MC1.12.1-1.2.4.jar                   | None                                     |
	| UCH   | rangedpumps                       | 0.5                          | rangedpumps-0.5.jar                                 | None                                     |
	| UCH   | reborncore                        | 3.8.9.300                    | RebornCore-1.12.2-3.8.9.300-universal.jar           | 8727a3141c8ec7f173b87aa78b9b9807867c4e6b |
	| UCH   | rebornstorage                     | 1.0.0                        | RebornStorage-1.12.2-3.1.0.50.jar                   | None                                     |
	| UCH   | refinedstorageaddons              | 0.3                          | refinedstorageaddons-0.3.jar                        | None                                     |
	| UCH   | resourceloader                    | 1.5.3                        | ResourceLoader-MC1.12.1-1.5.3.jar                   | d72e0dd57935b3e9476212aea0c0df352dd76291 |
	| UCH   | rftoolspower                      | 1.1.0                        | rftoolspower-1.12-1.1.0.jar                         | None                                     |
	| UCH   | sgextraparts                      | 1.4.1                        | SGExtraParts-1.12-1.4.1-34.jar                      | None                                     |
	| UCH   | soulshardstow                     | 1.12-2.7.6-56                | SoulShards-TOW-1.12-2.7.6-56.jar                    | None                                     |
	| UCH   | techreborn                        | 2.15.15.712                  | TechReborn-1.12.2-2.15.15.712-universal.jar         | 8727a3141c8ec7f173b87aa78b9b9807867c4e6b |
	| UCH   | thaumicjei                        | 1.2.1                        | ThaumicJEI-1.12.2-1.4.0-14.jar                      | None                                     |
	| UCH   | thermalcultivation                | 0.3.0                        | ThermalCultivation-1.12.2-0.3.0.7-universal.jar     | 8a6abf2cb9e141b866580d369ba6548732eff25f |
	| UCH   | thermaldynamics                   | 2.5.1                        | ThermalDynamics-1.12.2-2.5.1.14-universal.jar       | 8a6abf2cb9e141b866580d369ba6548732eff25f |
	| UCH   | tinkersaddons                     | 1.0.7                        | Tinkers' Addons-1.12.1-1.0.7.jar                    | None                                     |
	| UCH   | toastcontrol                      | 1.7.0                        | Toast Control-1.12.2-1.7.0.jar                      | None                                     |
	| UCH   | tombmanygraves2api                | 1.12.2-1.0.0                 | tombmanygraves2api-1.12.2-1.0.0.jar                 | None                                     |
	| UCH   | tombmanygraves                    | 1.12-4.1.0                   | TombManyGraves-1.12-4.1.0.jar                       | None                                     |
	| UCH   | traverse                          | 1.5.4                        | Traverse-1.12.2-1.5.4-61.jar                        | None                                     |
	| UCH   | universalmodifiers                | 1.12.2-1.0.9a                | valkyrielib-1.12.2-2.0.11a.jar                      | None                                     |
	| UCH   | waim                              | 1.0.0                        | WAIM-1.0.0.jar                                      | None                                     |
	| UCH   | reauth                            | 3.6.0                        | reauth-3.6.0.jar                                    | daba0ec4df71b6da841768c49fb873def208a1e3 |
	| UCH   | teslacorelib_registries           | 1.0.14                       | tesla-core-lib-1.12.2-1.0.14.jar                    | None                                     |

	Loaded coremods (and transformers): 
ShetiPhian-ASM (ShetiPhian-ASM-1.12.0.jar)
  shetiphian.asm.ClassTransformer
ForgelinPlugin (Forgelin-1.7.4.jar)
  
BedPatch (bedpatch-2.2-1.12.2.jar)
  com.mordenkainen.bedpatch.BedPatchASM
EnderCorePlugin (EnderCore-1.12.2-0.5.31.jar)
  com.enderio.core.common.transform.EnderCoreTransformer
  com.enderio.core.common.transform.SimpleMixinPatcher
LoadingPlugin (Quark-r1.4-125.jar)
  vazkii.quark.base.asm.ClassTransformer
Inventory Tweaks Coremod (InventoryTweaks-1.64-dev.jar)
  invtweaks.forge.asm.ContainerTransformer
AstralCore (astralsorcery-1.12.2-1.9.4.jar)
  
CTMCorePlugin (CTM-MC1.12-0.3.0.15.jar)
  team.chisel.ctm.client.asm.CTMTransformer
CoreMod (Aroma1997Core-1.12.2-2.0.0.0.b155.jar)
  
IELoadingPlugin (ImmersiveEngineering-core-0.12-82.jar)
  blusunrize.immersiveengineering.common.asm.IEClassTransformer
LoadingPlugin (ResourceLoader-MC1.12.1-1.5.3.jar)
  lumien.resourceloader.asm.ClassTransformer
Do not report to Forge! Remove FoamFixAPI (or replace with FoamFixAPI-Lawful) and try again. (foamfix-0.9.9.1-1.12.2-anarchy.jar)
  pl.asie.foamfix.coremod.FoamFixTransformer
	GL info: ' Vendor: 'ATI Technologies Inc.' Version: '4.5.13492 Compatibility Profile Context 22.19.677.257' Renderer: 'AMD Radeon HD 7800 Series'
	Pulsar/tconstruct loaded Pulses: 
		- TinkerCommons (Enabled/Forced)
		- TinkerWorld (Enabled/Not Forced)
		- TinkerTools (Enabled/Not Forced)
		- TinkerHarvestTools (Enabled/Forced)
		- TinkerMeleeWeapons (Enabled/Forced)
		- TinkerRangedWeapons (Enabled/Forced)
		- TinkerModifiers (Enabled/Forced)
		- TinkerSmeltery (Enabled/Not Forced)
		- TinkerGadgets (Enabled/Not Forced)
		- TinkerOredict (Enabled/Forced)
		- TinkerIntegration (Enabled/Forced)
		- TinkerFluids (Enabled/Forced)
		- TinkerMaterials (Enabled/Forced)
		- TinkerModelRegister (Enabled/Forced)
		- chiselIntegration (Enabled/Not Forced)
		- craftingtweaksIntegration (Enabled/Not Forced)
		- wailaIntegration (Enabled/Not Forced)

	Pulsar/natura loaded Pulses: 
		- NaturaCommons (Enabled/Forced)
		- NaturaOverworld (Enabled/Not Forced)
		- NaturaNether (Enabled/Not Forced)
		- NaturaDecorative (Enabled/Not Forced)
		- NaturaTools (Enabled/Not Forced)
		- NaturaEntities (Enabled/Not Forced)
		- NaturaOredict (Enabled/Forced)
		- NaturaWorld (Enabled/Not Forced)
		- craftingtweaksIntegration (Enabled/Not Forced)

	RebornCore: 
		Plugin Engine: 0
		RebornCore Version: 3.8.9.300
		Runtime Debofucsation 1
		RenderEngine: 0
	Ender IO: No known problems detected.

	!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
	!!!You are looking at the diagnostics information, not at the crash.       !!!
	!!!Scroll up until you see the line with '---- Minecraft Crash Report ----'!!!
	!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by Yamza** [comment]
See here: https://www.reddit.com/r/allthemods/comments/97t8zn/bug_gendustry_crashing_on_startup/
Please use pastebin for logs in the future :)

**Comment by sebi2306** [comment]
TL;DR Use the jar Launcher

**Comment by martmists** [comment]
@sebi2306 this is false as the error has been fixed in both extrautils2, forge AND minecraft itself.

**Comment by sebi2306** [comment]
Wrong, as it was a Mojang mistake that then got HOTFIXED by both XU2 and Forge 

> Apparently there's a XU2 crash involving languages, that's caused by changes in the Mojang launcher. It's being fixed on Mojang's side but I'll put out a quick hotfix in the mean time. - RWTema

> is it the language issue? It looks like Mojang broke the asset index 😐 - voxcpw
